packages/compass-indexes/src/components/search-indexes-modals/base-search-index-modal.spec.tsx

```diff
@@ -1,14 +1,24 @@
+import { ATLAS_SEARCH_TEMPLATES } from '@mongodb-js/mongodb-constants';
 import { expect } from 'chai';
 import { BaseSearchIndexModal } from './base-search-index-modal';
 import sinon from 'sinon';
 import type { SinonSpy } from 'sinon';
 
-import { render, screen, cleanup } from '@testing-library/react';
+import { render, screen, cleanup, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import React from 'react';
 import { getCodemirrorEditorValue } from '@mongodb-js/compass-editor';
 
+function normalizedTemplateNamed(name: string) {
+  const snippet =
+    ATLAS_SEARCH_TEMPLATES.find((t) => t.name === name)?.snippet || '';
+  // code mirror 'changes' the template placeholders, so let's do the same
+  // this regexp removes `tab` markers to their default value, for example:
+  // ${1:default} => default
+  return snippet.replace(/\${\d:([<>a-z ]+)}/gm, '$1');
```

Comment:
Shouldn't `\d` be `\d+` in case we ever have more than 9? I haven't really seen these, so no idea if that's possible. 

And maybe `[<>a-z ]` could just be `[^}]`? Wondering what happens with capitals and numbers and other punctuation.

Reply:
I'm not sure we can have more than 9, but let me apply the fixes, as they should be semantically the same, and it's already under tested, so we'll find out if it breaks.